chrissi/index.js

```diff
@@ -0,0 +1,60 @@
+//0
+
+let snackServings = 4;
+let guests = 16;
+
+console.log(snackServings, guests);
+
+// 1 ADD
+console.log(snackServings + guests);
+
+// 2 SUB
+console.log(snackServings - guests);
+
+// 3 MULT
+console.log(snackServings * guests);
+
+// 4 DIV
+console.log(snackServings / guests);
+console.log(guests / snackServings);
+
+// 5
+let drinks = 10;
+
+// (snackServings * guests) = 64
+
+console.log(64 / drinks);
+// (64 / 10) = 6.4
+
+let resultOne = 6.4;
+
+console.log(resultOne);
+
+// 6
+let monkeys = 15;
+let bananas = 9;
+console.log(monkeys / bananas);
+
+// 7
+
+let monkeyfights = 20;
+
+console.log((monkeys + bananas) * monkeyfights);
+
+let resultTwo = 480;
+console.log(resultTwo);
+
+// 8
+
```

Comment:
8th and 9th tasks answers are wrong! And the solution is :
8th- console.log(++monkeys)
9th- console.log(--bananas)

Reply:
we did NOT have this in the lesson

Reply:
Yeah I know, sorry for that!